ts/Data/DataPoolOptions.ts

```diff
@@ -34,7 +34,7 @@ export interface DataPoolOptions {
 export interface DataPoolConnectorOptions
 <T extends keyof DataConnectorTypes = keyof DataConnectorTypes> {
     id: string;
-    options: DataConnectorTypes[T]['prototype']['options'];
+    options: DeepPartial<DataConnectorTypes[T]['prototype']['options']>;
```

Comment:
Please use `DeepPartial<DataPoolConnectorOptions>`.

Reply:
It causes some errors.

Reply:
Maybe just remove it. I tested locally and everything works without DeepPartial. What was the reason to add DeepPartial?